### ISSUE TITLE
joker.repl Unavailable Outside REPL Mode

### ISSUE THREAD

**Issue by jcburley** [candid82/joker#143]
E.g.:

```
$ ./joker -e '(use (quote [joker.math]))'
$ ./joker -e '(use (quote [joker.walk]))'
$ ./joker -e '(use (quote [joker.template]))'
$ ./joker -e '(use (quote [joker.repl]))'
<joker.core>:3454:3: Eval error: open /home/craig/go/src/github.com/candid82/joker/joker/repl.joke: not a directory
Stacktrace:
  global <expr>:1:1
  core/use <joker.core>:3442:12
  core/apply <joker.core>:511:4
  core/apply__ <joker.core>:3340:9
  core/apply <joker.core>:509:4
  core/apply__ <joker.core>:3293:5
  core/with-bindings* <joker.core>:1396:7
  core/apply <joker.core>:507:4
  core/apply__ <joker.core>:3296:11
  load <joker.core>:3236:3
  core/load <joker.core>:3466:9
  core/with-bindings* <joker.core>:1396:7
  core/apply <joker.core>:507:4
  core/apply__ <joker.core>:3470:13
  core/load-file <joker.core>:3454:3
$
```

**Comment by candid82** [comment]
Fixed by 281f17ee27eca8bab11cd0c42d8cdf8274ef160a.

I resurrected your commit and refactored the code a little bit. I don't like the inconsistency (some std namespaces are loaded on startup and some are not), but I noticed that Clojure does that too. This should not be an issue from practical point of view.

**Comment by jcburley** [comment]
Thanks!
